README.md

```diff
@@ -110,11 +110,14 @@ There is no 'make install' yet. If you want to make an install
 package, run 'make distrib'.
 
 If you wish to use cmake to build Halide, the build procedure is:
+Note that ``LLVM_DIR`` should point to a directory containing ``LLVMConfig.cmake``.
+It is available in the LLVM build tree if LLVM was built with CMake and it is available
+from any LLVM build (3.5 and newer) that is installed.
 
     % mkdir build
     % cd build
     % LLVM_ROOT=/path/to/llvm3.5
-    % cmake -DLLVM_BIN=${LLVM_ROOT}/build/bin -DLLVM_INCLUDE="${LLVM_ROOT}/include;${LLVM_ROOT}/build/include" -DLLVM_LIB=${LLVM_ROOT}/build/lib -DLLVM_VERSION=35 ..
+    % cmake -DLLVM_DIR=${LLVM_ROOT}/build/share/cmake/
```

Comment:
@abadams What do you mean by clobber here?

If the user has a binary build they can also set ``LLVM_DIR`` to the directory containing ``LLVMConfig.cmake`` in the distribution they downloaded.